portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -55,11 +60,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Get comments from datastore, and add numCommentsRequested number of comments to comments list
     PreparedQuery results = datastore.prepare(commentsQuery);
-    List<String> comments = new ArrayList<>();
+    List<Comment> comments = new ArrayList<>();
```

Comment:
hmm I don't see this new `Comment` class defined anywhere; did you possibly forget to `git add` it?

Reply:
Whoops, I'll add that